x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_tooltip.tsx

```diff
@@ -46,11 +48,29 @@ export class DrawTooltip extends Component<Props, State> {
   }
 
   componentWillUnmount() {
+    this._isMounted = false;
     this.props.mbMap.off('mousemove', this._updateTooltipLocation);
     this.props.mbMap.off('mouseout', this._hideTooltip);
     this._updateTooltipLocation.cancel();
   }
 
+  _hideTooltip = () => {
+    this._updateTooltipLocation.cancel();
+    this.setState({ isOpen: false });
+  };
+
+  _updateTooltipLocation = _.throttle(({ lngLat }) => {
+    const mouseLocation = this.props.mbMap.project(lngLat);
+    if (!this._isMounted) {
```

Comment:
Not async, but occasionally see error messages due to throttled callback executing after component unmounted.